README.md

```diff
@@ -1 +1,20 @@
 # hw01
+
+My name is Mackenzie Whitfield.  
+I moved to Chicago from San Diego two weeks ago. I **love** the beach. My hobbies also include hiking, traveling, reading, and watching movies. Up until now, I have only lived in California.  
+
+# Where I have Lived 
```

Comment:
The list function here is really clear and clean, a wonderful way of showing the cities!